core/domain/learner_progress_services.py

```diff
@@ -1218,13 +1357,16 @@ def _get_filtered_partially_learnt_topic_summaries(
     """
     nonexistent_partially_learnt_topic_ids = []
     filtered_partially_learnt_topic_summaries = []
-    topic_rights = topic_fetchers.get_multi_topic_rights(topic_ids)
+    topic_rights_with_none = topic_fetchers.get_multi_topic_rights(topic_ids)
```

Comment:
Is there a possibility that `topic_rights_with_none` can contain `None`?

I see we are asserting `topic_rights_with_none[index]` is not `None` below. If we are sure that `topic_rights_with_none` won't contain `None`, then wouldn't it be better to change the variable name?

Ditto elsewhere.

Reply:
Changed !